src/chat/GalleryScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useEffect, useRef, useState } from 'react';
+import {
+  View,
+  Text,
+  FlatList,
+  StyleSheet,
+  TouchableOpacity,
+  Dimensions,
+  ImageRequireSource,
+} from 'react-native';
+import FastImage from 'react-native-fast-image';
+import { FirestoreServices } from '../services/firebase';
+import { MessageTypes, type MediaFile } from '../interfaces';
+import ThumbnailVideoPlayer from './components/camera/ThumbnailVideoPlayer';
+import SelectedViewModal from './components/camera/SelectedViewModal';
+import { GalleryType } from '../interfaces/gallery';
```

Comment:
Re-organize file export and import if possible, export them all in `index` file so we can import them together

Reply:
ok I updated